### PR TITLE
Reframe/image component

### PR DESCRIPTION
**Pull request by Corei13**
image

### CODE REVIEW

File: lib/base/Image.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+});
+
+type Props = $.Static<typeof $Props>;
+
+const Image = ({ height, width, src }: Props) => (
+  <NextImage
+    width={100}
+    height={100}
+    src={
+      "https://cdn.bhdw.net/im/cool-white-cat-with-round-sunglasses-wallpaper-54804_w635.jpg"
```
Comment (Corei13) [inline lib/base/Image.tsx:19]: we should have width={width} etc instead of hardcoded values

File: lib/app/code.json [hunk 2]
```diff
@@ -21,6 +21,9 @@
       "type": "Button",
       "children": ["1-1-1-2"]
     },
+    "1-1-3": {
+      "type": "Image"
```
Comment (Corei13) [inline lib/app/code.json:25]: instead, add widht, height and src here - ask ranit for help

**Review by Corei13 — COMMENTED** [review]
added some comments + resolve the conflicts with ranit

File: lib/app/code.json [hunk 3]
```diff
@@ -22,7 +22,11 @@
       "children": ["1-1-1-2"]
     },
     "1-1-3": {
-      "type": "Image"
+      "type": "Image",
+      "props": {
+        "width": 100,
+        "height": 100
```
Comment (ranit43) [inline lib/app/code.json:28]: add image src from here.

File: lib/base/Image.tsx [hunk 4]
```diff
@@ -12,12 +13,6 @@ const $Props = $.Record({
...
 
 const Image = ({ height, width, src }: Props) => (
-  <NextImage
-    width={100}
-    height={100}
-    src={
-      "https://cdn.bhdw.net/im/cool-white-cat-with-round-sunglasses-wallpaper-54804_w635.jpg"
-    }
-  />
+  <NextImage width={width} height={height} src={sampleImage} />
```
Comment (ranit43) [inline lib/base/Image.tsx:16]: we should use the **image source** with the `src` prop received for the component. @ashfaquetuhin

**Pull request merged**